src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -196,6 +196,8 @@ internal bool IsScriptFileDetermined(FunctionMetadata functionMetadata)
 
         private void LoadCustomProviderFunctions(List<FunctionMetadata> functionMetadataList)
         {
+            _logger.FunctionMetadataProviderParsingFunctions();
```

Comment:
We make the same logging call from HostFunctionMetadataProvider as well. But the "Source" column will be able to help us to distinguish it from that one and this one.